### ISSUE TITLE
License as MIT

### ISSUE THREAD

**Issue by christophsax** [ropensci/tsbox#230]
This may be more useful for such an infrastructure package.

Dependency data.table is fine, but we need a substitute for anytime.

**Comment by moodymudskipper** [comment]
with a bit of luck anytime could become MIT?  https://github.com/eddelbuettel/anytime/issues/136
cc @Layalchristine24

**Comment by christophsax** [comment]
@moodymudskipper do we really need this? 

AI tells me:

> Your MIT license is fine regardless of your dependencies' licenses. The MIT license is one of the most permissive and compatible licenses available. Just make sure to include the MIT license text in your LICENSE or LICENSE.md file and reference it properly in your DESCRIPTION file.

**Comment by eddelbuettel** [comment]
That is also what I said in https://github.com/eddelbuettel/anytime/issues/136.  You are not avoiding the `gcc` compiler or Linux operating system or `bash` shell ... (I could go on) when you are building this package either.  It is irrelevant what license a (reverse or direct) dependee is under: you are authors of a self-contained unit of code, can choose its license and then obey by its obligation.  And there is no such (falsely imagined) 'MIT package cannot depend on or use GPL-* package'.  You cannot _copy GPL code into a MIT-licensed package_ but that is a very different kettle of fish.

So with that the [original assessment above](https://github.com/ropensci/tsbox/issues/230#issue-2600333768) namely

> Dependency data.table is fine, but we need a substitute for anytime.

seems not to be relevant.  Carry on relicensing as you see fit.

PS Also, of course, R itself.  See `licence()` or `license()`.

**Comment by moodymudskipper** [comment]
So dependencies are not an issue, but we do need the consent of all contributors, even if they authored minor contributions.
If they contributed on behalf of an organization it complicates things a bit. 

* I'll ask for explicit consent below in this thread
* For those that don't answer we should send emails
* For those that don't answer emails or refuse the change we can either : 
  * Use dual licensing, with some parts of the code licensed differently, but it sounds like a headache
  * Remove their contributions and rewrite them ourselves

**Comment by moodymudskipper** [comment]
Hi everyone,

We are planning to relicense this project from the GNU General Public License (GPL) to the MIT License, to allow for broader use and simpler integration.

Since this project includes contributions from multiple authors, I need explicit permission from each contributor to relicense their contributions under the MIT license. These are currently : 

@christophsax 
@Thu-G-Hoang 
@maelle ✅ 
@mbannert 
@karoliskoncevicius 
@Layalchristine24 ✅ 
@will-the-wiz  
@krlmlr ✅ 

If you have contributed code on your own behalf and agree to give permission, please reply to this issue with the following sentence:

> "I consent to relicense my contributions to this project under the MIT License."

If you contributed on behalf of an organisation please inform us and if you can't easily get their own written approval we'll find another solution.

The full text of the MIT license can be found here: https://opensource.org/licenses/MIT

If you have any questions or concerns, feel free to raise them here.

Thanks in advance for your help!

**Comment by krlmlr** [comment]
I consent to relicense my contributions to this project under the MIT License.

**Comment by moodymudskipper** [comment]
Since I might myself get on the contributor list before the switch I'll do it too preemptively:

I consent to relicense my contributions to this project under the MIT License.

**Comment by Layalchristine24** [comment]
I consent to relicense my contributions to this project under the MIT License.

**Comment by maelle** [comment]
I consent to relicense my contributions to this project under the MIT License.

**Comment by karoliskoncevicius** [comment]
I consent to relicense my contributions to this project under the MIT License.

**Comment by Thu-G-Hoang** [comment]
I consent to relicense my contributions to this project under the MIT
License.

**Comment by christophsax** [comment]
@mbannert, @will-the-wiz?

MIT license was asked for by some users, and I don't see a reason for not to relicense.

**Comment by mbannert** [comment]
Sure, I am also happy to agree. Plus, I can speak for Will, too. he is an alter ego of mine I use in the teaching. Not sure how he ended up a contributor of tsbox, possibly some mail address hiccup on my side. Sorry for the delay and mess. 

TLDR; Go!
